cspell/Makefile

```diff
@@ -0,0 +1,26 @@
+# These commands must be run from the VictoriaMetrics repository root.
+
+cspell-build:
+	@docker build cspell \
+		--tag cspell
+
+cspell-check:
+	@CMD="cspell --no-progress '**'" $(MAKE) cspell-run-command
+
+cspell-run-command:
+	@docker run \
+		--entrypoint /bin/sh \
+		--mount type=bind,src=".",dst=/victoriametrics \
+		--rm \
+		--tty \
+		cspell -c "$(CMD)"
+
+cspell-update: cspell-build
```

Comment:
Would you mind clarifying when this and `spellcheck-install` should be executed?

Reply:
I've added comments for each target. Please check if that's what you expected.